lib/flipper/dsl.rb

```diff
@@ -163,6 +163,16 @@ def feature(name)
       })
     end
 
+    def preload(names)
+      features = names.map { |name| feature(name) }
+      @adapter.get_multi(features)
+      features
+    end
+
+    def preload_all
```

Comment:
I'm on the fence about encouraging this as it could become equally as damaging as getting things one by one, but I can always add a caveat to the caveats.md file about being careful to not have too many features while using preload_all.